### PR TITLE
Make storageclass paramter case in-sensitive

### PR DESCRIPTION
**Pull request by leakingtapan**
**Is this a bug fix or adding new feature?**
Fixes: #278 

/cc @bertinatto

### CODE REVIEW

File: pkg/driver/controller.go [hunk 1]
```diff
@@ -99,8 +100,35 @@ func (d *controllerService) CreateVolume(ctx context.Context, req *csi.CreateVol
...
+			}
+		case EncryptedKey:
+			if value == "true" {
+				isEncrypted = true
+			} else {
+				isEncrypted = false
+			}
+		case KmsKeyIdKey:
+			kmsKeyId = value
+		}
```
Comment (bertinatto) [inline pkg/driver/controller.go]: Shouldn't we explicitly default to an `codes.InvalidArgument` error if we get something unexpected?

Do you know what the in-tree plugin does in this case? If it simply ignores the parameter, we may do the same.

File: pkg/driver/controller.go [hunk 2]
```diff
@@ -99,8 +100,35 @@ func (d *controllerService) CreateVolume(ctx context.Context, req *csi.CreateVol
...
+	var (
+		fsType      string
+		volumeType  string
+		iopsPerGB   int
+		isEncrypted bool
+		kmsKeyId    string
+	)
+
+	for key, value := range req.GetParameters() {
+		switch strings.ToLower(key) {
```
Comment (bertinatto) [inline pkg/driver/controller.go]: Just to clarify, do we support a lower case `fsType` in-tree?  What about the value?

File: pkg/driver/controller.go (hunk 1 above)
Comment (leakingtapan) [reply pkg/driver/controller.go]: It does return errors: https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/awsebs/aws_util.go#L180

Updated

File: pkg/driver/controller.go (hunk 2 above)
Comment (leakingtapan) [reply pkg/driver/controller.go]: We do support lower case `fstype` in-tree as the key, that's [this PR](https://github.com/kubernetes/kubernetes/issues/75941) fixes for in-tree.

**Comment by bertinatto** [comment]
Nice fix, good job on keeping in-tree and driver synced.

The changes LGTM, but I'd like to see a test for this. If you want to get this in ASAP, we can merge this and can create an Issue for the current cycle.

**Comment by leakingtapan** [comment]
@bertinatto added unit test. PTAL

**Pull request merged**